### ISSUE TITLE
PAL2 does not seem to launch at all

### ISSUE THREAD

**Issue by uwidev** [POE-Addon-Launcher/PAL2#11]
When starting the program, nothing happens. Checking the task manager, when trying to open the program I can see it appear for a second before it magically disappears. The PAL_Logger.log doesn't have anything in it, if it's supposed to dump something.

I'm on Windows 10.

edit - the PAL folder doesn't exist under Appdata either

**Comment by bighoho123** [comment]
I have having the same issue win10.

**Comment by JasonCraneAdkins** [comment]
Same happen  to me after reinstalling my PC.
I forgot to install java, by any case did you?

**Comment by Smargoos** [comment]
I have the same problem, but for me it is fixed by launching it as an administrator.

Edit: Ok so I did some testing and the problem for me seems to be trying to start it from the search bar. If I try to open it in the folder it is unzipped in it has no problems. I think the program tries to create the log in whatever folder it is launched from and in the case of the search bar that is system32. I found 2 pal2 logs on my pc one in the unzipped folder and the other system32. In fact I was able to create a third log by starting the program from cmd that was in a different folder. The reason for the crash is probably that it isn't allowed to change/add things to system32 or the folder you have unzipped it into if that is protected without administrator permissions.
